ember/app/ui/cases/detail/index/template.hbs

```diff
@@ -103,12 +145,43 @@
           {{t "documents.accountingExport"}}
         </UkButton>
       {{/if}}
-
-      {{#if (not (or this.isEditable this.isNotSubmitted (can "exportForAccounting case")))}}
-        <span>{{t "documents.noActions"}}</span>
-      {{/if}}
     {{/if}}
   </div>
 </div>
 
 <hr />
+
+<div id="modal-access" uk-modal>
+  <div class="uk-modal-dialog uk-modal-body">
+    <button type="button" class="uk-modal-close-default" uk-close></button>
+
+    <h2 class="uk-modal-title">
+      {{t "documents.accesses.add"}}
+    </h2>
+
+    <div class="uk-margin">
+      <ValidatedForm
+        @model={{this.newRow}}
+        @on-submit={{perform this.saveRow}}
+        as |Form|
+      >
+        <Form.input
+          @label={{t "documents.accesses.email"}}
+          @name="email"
+        />
+
+        <div class="uk-text-right">
+          <UkButton
+            {{on "click" (perform this.cancelRow)}}
+            class="uk-modal-close"
+          >
+            {{t "global.cancel"}}
+          </UkButton>
+          <Form.submit
+            @label={{t "global.save"}}
+          />
+        </div>
+      </ValidatedForm>
+    </div>
+  </div>
+</div>
```

Comment:
What is the reason for not using `<UkModal`?

Reply:
Forgot it existed, changed